addStyles.js

```diff
@@ -21,9 +21,7 @@ var stylesInDom = {},
 	styleElementsInsertedAtTop = [];
 
 module.exports = function(list, options) {
-	if(typeof DEBUG !== "undefined" && DEBUG) {
-		if(typeof document !== "object") throw new Error("The style-loader cannot be used in a non-browser environment");
```

Comment:
Yeah, better leave. It should be documented visibly that you are meant to use **style-loader** in browser environment only.